packages/shared/src/styles/themes.scss

```diff
@@ -175,5 +177,7 @@ $themes: (
         won_gradient_color               : $COLOR_DARK_GREEN_GRADIENT,
         won_color                        : $COLOR_GREEN_1,
         loss_color                       : $COLOR_RED,
+        border_line                      : $COLOR_WHITE,
+        disable_color                    : $COLOR_LIGHT_GRAY_1,
```

Comment:
Will it be visible in dark mode?

Reply:
I we don't have dark mode in bot?

Reply:
you can try var(--border-normal)
and var(--text-disabled)